include/aca_ovs_control.h

```diff
@@ -0,0 +1,108 @@
+// Copyright 2019 The Alcor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ACA_OVS_CONTROL_H
+#define ACA_OVS_CONTROL_H
+
+#define IPTOS_PREC_INTERNETCONTROL 0xc0
+#define DSCP_DEFAULT (IPTOS_PREC_INTERNETCONTROL >> 2)
+#define STDOUT_FILENO   1   /* Standard output.  */
+
+#include <openvswitch/ofp-errors.h>
+#include <openvswitch/ofp-packet.h>
+#include <string>
+
+// OVS monitor implementation class
+namespace aca_ovs_control
+{
+class ACA_OVS_Control {
+  public:
+  static ACA_OVS_Control &get_instance();
+
+  int control();
+  int monitor(const char *bridge, const char *opt);
+  int packet_out(const char *bridge, const char *opt);  
+  int dump_flows(const char *bridge, const char *opt);
+  //int get_packet_in(const char *bridge); 
```

Comment:
remove the commented line if it is not needed

Reply:
the line has been removed.